File: javascript/grammer/10_list_comprehension.js
const evenSquares = Array.from({ length: 10 }, (_, i) => i + 1)
  .filter((x) => x % 2 === 0)
  .map((x) => x * x);

console.log('짝수 제곱 배열:', evenSquares);
console.log('원소 개수:', evenSquares.length);

File: cpp/grammer/10_list_comprehension.cpp
#include <iostream>
#include <vector>

int main() {
    std::vector<int> even_squares;
    for (int x = 1; x <= 10; ++x) {
        if (x % 2 == 0) even_squares.push_back(x * x);
    }

    std::cout << "짝수 제곱 리스트: ";
    for (int x : even_squares) std::cout << x << ' ';
    std::cout << "\n원소 개수: " << even_squares.size() << "\n";
}
